UOP1_Project/Assets/Scripts/CameraManager.cs

```diff
@@ -1,34 +1,34 @@
-﻿using System.Collections;
-using System.Collections.Generic;
-using UnityEngine;
 using Cinemachine;
-using System;
+using UnityEngine;
 
-public class CameraManager : MonoBehaviour
+namespace Assets.Scripts
```

Comment:
Namespaces should not start with `Assets.Scripts` imo, the root for namespaces should be the Scripts folder itself. This class because of that shouldn't have a namespace yet but technically it should be moved to a sub directory `Camera` and then get the namespace `Camera` as well

Reply:
Also to build on top of that, once we have a name for the game that should become the root namespace in order to properly differentiate it from the other packages and libraries that are being used. It might be best to leave the namespaces out for now and think about those later once we have a name for the game

Reply:
This has been changes to `UOP1` as the root namespace :)